### DISCUSSION TITLE
scritte colorate?

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#54] [Domande & Rispsote]
in questa immagine mi accingo a parlare con la guardia di sicurezza nel sotteraneo (jack-tutorial).

la domanda è : quelle scritte colorate in inglese, si possono tradurre? e come?

forse è una immagine grafica e bisogna modificare quella?

una luce por favor.

![scritte colorate](https://user-images.githubusercontent.com/96891196/149784702-804831db-0c22-4c4d-98c1-a37bf18bde93.jpg)

**Comment by NarakuITA** [comment]
Si è un immagine. Non ho idea di dove sia.

**Reply by crazyLillo** [reply]
io una mezza idea c'è l'ho, ma ci vuole un programma da scaricare, poi aprire uno ad uno finche non si trova quell immagine da modificare
ma poi come la si salverebbe nello stesso formato, lo accetterebbe? so che ci vogliono i giusti programmi per farlo bene, ma non lo so quanto può valere la pena...

Intanto fatto questo, datemi un opinione, Please.

![meglio](https://user-images.githubusercontent.com/96891196/149847604-b2adfbac-7e0d-43e3-a01d-5a3cbcc93fc3.jpg)

**Reply by crazyLillo** [reply]
**datemi un opinione e se devo cambiare qualcosa.**

**Reply by NarakuITA** [reply]
Non ho esperienza. Non saprei dirti. Per ora è meglio tralasciare, se si vuole cambiare l'immagine.  Tuttavia inserire una piccola descrizione per me andrebbe benissimo!

**Reply by crazyLillo** [reply]
infatti la descrizione lo messa su nel riquadro con le scritte bianche!

Io non volevo arrivare ad usare SDK, perchè è quello che ci vuole per fare quel tipo di modifica, anzi un sacco di modifiche si possono fare.
ma se comincio ad usare l'SDK, poi non credo di continuare con la traduzione perchè mi ci perderei con piacere.
cosa che mi riservo in futuro.

Quindi se così come ho fatto potrebbe andare bene. la lasciamo così.

**Reply by tesivo** [reply]
L'idea della legenda nel riquadro con le scritte in bianco è ottima. Per ora va bene così. Eventualmente in futuro si applicherà come si deve anche questa modifica.

**Comment by crazyLillo** [comment]
L'importante è far capire al giocatore le dinamiche di gioco.

quelli sembrano essere file grafici e per modificarli ci vuole SDK (credo).
